examples/metrics/filtering/se/src/main/java/io/helidon/examples/metrics/filtering/se/Main.java

```diff
@@ -100,7 +100,7 @@ static void routing(HttpRouting.Builder routing, Config config, MetricsConfig.Bu
         MetricsObserver metrics = MetricsObserver.builder()
                 .metricsConfig(metricsConfigBuilder)
                 .build();
-        GreetService greetService = new GreetService(config, meterRegistry);
+        GreetService greetService = new GreetService(meterRegistry);
```

Comment:
Re: line 97 above - For consistency and emphasis (of not passing config as parameters, for the most part) do we want to remove the `config` parameter from `routing`?

Reply:
Done